### PR TITLE
update some log level from Debug to Information

### PR DESCRIPTION
**Pull request by sky1045**
SSIA

### CODE REVIEW

File: Libplanet/Blockchain/Renderers/LoggedActionRenderer.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public class LoggedActionRenderer<T> : LoggedRenderer<T>, IActionRenderer<T>
         public LoggedActionRenderer(
             IActionRenderer<T> renderer,
             ILogger logger,
-            LogEventLevel level = LogEventLevel.Debug
+            LogEventLevel level = LogEventLevel.Information
```
Comment (area363) [inline Libplanet/Blockchain/Renderers/LoggedActionRenderer.cs:47]: Wouldn't this prevent debug messages from being logged as stdout? I might have misunderstood but I thought we were logging all the debug messages on the stdout but filter info logs to CW.

Comment (greymistcube) [reply Libplanet/Blockchain/Renderers/LoggedActionRenderer.cs:47]: I suspect this breaks tests. 🤔

**Review by area363 — APPROVED** [review]

**Comment by greymistcube** [comment]
Lemme push some additional changes before merging. 🙃

**Comment by sky1045** [comment]
by the way, i'm not sure why some tests fail even if the change i made is only for log levels 🤷‍♂️

**Comment by greymistcube** [comment]
I've tried to be a bit more selective on which logs get elevated to `Information` level.

**Review by greymistcube — APPROVED** [review]

**Pull request merged**